app.js

```diff
@@ -50,36 +56,48 @@ function showButton() {
     submitButton.classList.add('show-button');
 }
 
-function startPlaying() {
-    
+function startPlaying() {     
+    if(randomNumber === 0) {
```

Comment:
as not array (as I showed on the lesson) was chosen let's use another approach (but only for learning, current solution is proper ;) ). 
Try to use `switch` statement :) 

I guess, that `startPlaying` will be not a good name, as this function does not start a game, but is choosing the value for computer. maybe `getComputerValue` ?